src/Controller/UrlGeneratorController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use App\Repository\UrlBaseRepository;
+
+class UrlGeneratorController extends AbstractController
+{
+    /**
+     * @Route("/url/generator/result", name="url_generator_result")
+     */
+    public function index(UrlBaseRepository $urlBaseRepository): Response
+    {
+        return $this->render('url_generator/result.html.twig', [
+            'url_generator' => $urlBaseRepository->findBy([], ['id'=>'DESC'],1),
```

Comment:
1. teraz wyświetlasz ostatni dodane url, to może być problem jeśli dwóch użytkowników niemal jednocześnie coś wygenereuje - moze się okazać, że zostanie im przydstawiony wynik nie-ich zapytania. Przy przekierowaniu na url_generator_result po prostu dodaj jako argument id wygenerowanego URLa
2. Ta klasa spokojnie może być połączona z UrlBaseController.php